lib/board.rb

```diff
@@ -1,14 +1,14 @@
 class Board
-  def initialize
-    @board = [
-      [nil, nil, nil],
-      [nil, nil, nil],
-      [nil, nil, nil]
-    ]
+  def initialize(width:)
+    @board = generate_empty_board(width: width)
   end
 
   def state
-    @board
+    board
+  end
+
+  def width
+    board.length
```

Comment:
What are the benefits of wrapping board.length into `#width` method instead of using the `width` argument from initializer with the help of attr_reader?